### PR TITLE
[15.0] [IMP] document_url: upload url from upload activity

### PR DESCRIPTION
**Pull request by EmilioPascual**
If you create upload activity, button to upload url appears

@Moduon MT-3175 @yajo @shide @rafaelbn

### CODE REVIEW

File: document_url/static/src/xml/url.xml [hunk 1]
```diff
@@ -54,4 +54,19 @@
...
+        <xpath
+            expr="//div[hasclass('o_Activity_tools')]//t[2]/FileUploader"
+            position="after"
+        >
+            <button
+                type="button"
+                class="o_Activity_toolButton o_Activity_uploadButton btn btn-link"
+                t-on-click="_onAddUrl"
+            >
+                <i class="fa fa-plus-square" /> Add URL</button>
```
Comment (yajo) [inline document_url/static/src/xml/url.xml]: I think this is clearer.

```suggestion
                <i class="fa fa-link" /> Link document by URL</button>
```

**Review by yajo — CHANGES_REQUESTED** [review]
I noticed that when you upload a document, the activity is set as done. However, when you add an URL, it isn't. I think the workflow should be the same, isn't it?

**Review by yajo — APPROVED** [review]
Good now. Please squash to merge. Thanks!

**Review by rafaelbn — APPROVED** [review]
Great improvement 👍🏼 😄 

Please squash and if possible add a simple GIF to show new feature

**Comment by EmilioPascual** [comment]
Squash done.
Gif:
ezgif com-video-to-gif (1)

**Pull request merged**